privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -29,6 +29,9 @@ myApp.controller("tokenController", function (TokenFactory, ConfigFactory,
     $scope.loggedInUser = AuthFactory.getUser();
     $scope.selectedToken = {serial: null};
 
+    // listen to the reload broadcast
+    $scope.$on("piReload", $scope.get);
```

Comment:
I think this needs to be moved below, as ``$scope.get`` is not yet defined here (or we do ``function () { $scope.get() }``).

Reply:
angular does not care about this. Works fine this way, since the listener is called late after the complete controller definition.